README.md

```diff
@@ -32,33 +32,20 @@ Some of the targetted applications are:
 ## Overview 
 
 Kubernetes Local persistent volumes allows users to access local storage through the
-standard PVC interface in a simple and portable way.  The PV contains node
+standard PVC interface in a simple and portable way. The PV contains node
 affinity information that the system uses to schedule pods to the correct
-nodes.
-
-OpenEBS Dynamic Local PVs extends the capabilities provided by the Kubernetes Local PV
-by making use of the OpenEBS Node Storage Disk Manager (NDM), the significant
-differences include:
-- Users need not pre-format and mount the devices in the node.
-- Supports Dynamic Local PVs - where the devices can be used by CAS solutions
-  and also by applications. CAS solutions typically directly access a device.
-  OpenEBS Local PV ease the management of storage devices to be used between
-  CAS solutions (direct access) and applications (via PV), by making use of
-  BlockDeviceClaims supported by OpenEBS NDM.
+nodes. Features:
+
 - Supports using hostpath as well for provisioning a Local PV. In fact in some
   cases, the Kubernetes nodes may have limited number of storage devices
   attached to the node and hostpath based Local PVs offer efficient management
   of the storage available on the node.
 
 ## Kubernetes Compatibility Matrix
 
-|          | Kubernetes <= 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 | Kubernetes 1.19 | Kubernetes 1.20 | Kubernetes 1.21 | Kubernetes 1.22 | Kubernetes 1.23 | Kubernetes 1.24 | Kubernetes 1.25 | Kubernetes 1.26 | Kubernetes 1.27 |
-|----------|--------------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|
-| `v3.3.x` | ✕                  | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               |
-| `v3.4.x` | ✕                  | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               |
-| `v3.5.x` | ✕                  | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               |
-| `HEAD`   | ✕                  | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               |
-
+|          | Kubernetes <= 1.19 | Kubernetes 1.20 | Kubernetes 1.21 | Kubernetes 1.22 | Kubernetes 1.23 | Kubernetes 1.24 | Kubernetes 1.25 | Kubernetes 1.26 | Kubernetes 1.27 |
```

Comment:
1.19 should be the minimum.
We should have a `<= 1.18` table entry with the `✕`
1.19 should have a `✓`

Reply:
Also it should list 3.4, 3.5 and 4.0

Reply:
We don't want to list 3.4 and 3.5 here right? They have different scopes and features.

Reply:
Is the version change for openebs or localpv? If it's for localpv, I want us to consider releasing a 3.6 or whatever is the next version. Basically, I want us to not tie the openebs umbrella version with the localpv hostpath version.

Reply:
This change is for localpv. Since its a breaking change we decided to bump the major version to 4.x, which happens to have coincided with openebs version. LocalPV versions are anyhow independent of umbrella.

Reply:
Ok, makes sense.